tests/PageObjects/RemoteMic/RemoteMicMainPage.ts

```diff
@@ -15,7 +16,31 @@ export class RemoteMicMainPagePO {
     await this.playerNameInput.fill(name);
   }
 
+  public async expectPlayerNameToBe(playerName: string) {
+    await expect(this.playerNameInput).toHaveValue(playerName);
+  }
+
   public async goToSongList() {
     await this.page.getByTestId('menu-song-list').click();
   }
+
+  public async goToSettings() {
+    await this.page.getByTestId('menu-settings').click();
+  }
+
+  public async expectPlayerToBeAssigned(micColor: string) {
```

Comment:
Instead of just `string`, it should specify literal values that are accepted, `'blue' | 'red'` etc.